src/core/Tooltip/TooltipContent/TooltipContent.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {
+  Component,
+  forwardRef,
+  RefObject,
+  ReactNode,
+  MouseEvent,
+  KeyboardEvent,
+} from 'react';
+import { default as styled } from 'styled-components';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { HtmlButton } from '../../../reset/HtmlButton/HtmlButton';
+import { Icon } from '../../Icon/Icon';
+import { baseStyles } from './TooltipContent.baseStyles';
+
+const baseClassName = 'fi-tooltip';
+
+const tooltipContentClassNames = {
+  closeButton: `${baseClassName}_close-button`,
+  closeButtonIcon: `${baseClassName}_close-button_icon`,
+};
+
+interface TooltipContentProps {
```

Comment:
Should we extend `HtmlDiv` in the interface?

Reply:
when extending, it might be worthwhile to make sure potentially breaking layout props are left out as the whole structure relies on being inline + block / inline-block elements.

Reply:
Changed it now to extend `HtmlDiv`.

Reply:
This is now changed.